### DISCUSSION TITLE
I'm getting 404 on a Cashier call to my webhook controller: checkout.session

### DISCUSSION THREAD

**Discussion by rabol** [laravel/framework#37777] [Q&A]
Hi

I'm using the latest version of Laravel 8 and Cashier and I have created / extended the webhookcontroller to handle more callbacks from Stripe, but...

For some reason I can see that there is a  404 when Stripe do a post to the route for the checkout.session event

In my version of the web hook controller, I log all events that does not have a handleXXX method

Any hints on what I am missing ?

**Comment by browner12** [comment]
I would recommend listing your **exact** versions of the Framework and Cashier, since *latest* can change quickly.

I'm not sure if this is related, but I also got a random 404 error for a Twilio webhook this morning, that I cannot replicate.

```php
Route::post('whisper', [CallController::class, 'whisperMessage'])->name('whisper');
```

I'm chalking it up as a fluke for now, but would be interested to hear if others have had this issue recently.

**Reply by rabol** [reply]
Cashier is version 12.14.1
Laravel is version 8.48.0

I think I have found it: in the Stripe event log it looks like "object": "checkout.session",

but the handleWebhook() translate it to handleCheckoutSessionCompleted, and in that method I had a bug
